docs/the-new-architecture/use-app-template.md

```diff
@@ -1,7 +1,134 @@
 ---
 id: use-app-template
-title: Using the New App Template
+title: Creating a New Architecture App
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+import RemoveGlobalCLI from '.././\_remove-global-cli.md';
+
+This page will help you create a new React Native app that uses the New Architecture.
+
+## Development environment
+
+Before continuing, make sure you've followed all the steps in [Setting up the
+development environment](getting-started.md), under the **React Native CLI
+Quickstart** tab.
+
+If following the setup guide, stop when you reach the section **Running your React Native
+Application**, and resume following this guide.
+
+**Note**: The New Architecture cannot currently be enabled in an Expo app.
+
+## Creating a new application
+
+<RemoveGlobalCLI />
+
+If you already have your development environment set up, create a new React Native project from the template:
+
+```shell
+npx react-native init AwesomeProject
+```
+
+If specifying a versioned template, use version 0.68 or later.
```

Comment:
```suggestion
**Note:** the New Architecture is available from version 0.68 or later.
```

I found personally confusing the _If specifying a version_. What does that mean? Should we push for using a versioned template? Or in case a user wants to use a versioned template, we are suggesting using version 0.68 or later?